src/SASViyaApiClient.ts

```diff
@@ -499,6 +506,35 @@ export class SASViyaApiClient {
           accessToken
         )
         console.log(`Parent Folder "${newFolderName}" successfully created.`)
+        parentFolderUri = `/folders/folders/${parentFolder.id}`
+      } else if (isForced && accessToken && !this.isForceDeploy) {
+        this.isForceDeploy = true
+
+        await this.deleteFolder(parentFolderPath, accessToken)
```

Comment:
need to have condition here, if unable to deleteFolder for any reason. 
we can show error cannot move/delete current folder.

Reply:
improved error handling in moveFolder function